routes/donations.js

```diff
@@ -14,6 +15,19 @@ router.get('/', async (req, res) => {
   }
 });
 
+router.get('/verify', async (req, res) => {
+  const { email, donationId } = req.body;
+  const donation = await db.query(`SELECT email FROM donations WHERE id = $(donationId);`, {
+    donationId,
+  });
+
+  if (donation[0].email === email) {
+    res.status(200).send('success');
+  } else {
+    res.status(500).send('donation id and email do not match');
```

Comment:
change this to follow the standard backend route format
```
(req, res) => {
  try {

  } catch (err) {
    ...
  }
}
```

Reply:
instead of using statuses to determine whether or not the email/donationId combination is correct, just send a boolean value